chemdyg/templates/chemdyg_STE_flux_diags.bash

```diff
@@ -234,6 +218,17 @@ else:
      yS_ann[0] = 'nan'
      yS_ann[1:12] = y[0:11]*1.E-9*12
 
+STE_month = xr.DataArray(np.array(STE_2D), coords=[month[1::],ncol], dims=["month","ncol"])
+STE_month_mean= np.zeros((12,len(ncol)))
+for i in range(12):
+    if len(time) == 12:
+        STE_month_mean[0,:] = 0
+        STE_month_mean[1::,:] = 1.e3*12*STE_month
```

Comment:
Multiplying by 12 ignores that different months have different days. Same for other lines, like line 219 above and other places. Should fix this. Maybe in a separate PR after this one is merged.

Reply:
@tangq Different days in different months have been considered in earlier
```dt = monthrange(2001,month[i])[1]*3600*24```
*12 is to covert the unit from per month to per year.

Reply:
Good to know it's been calculated. Would you add some comments to make it clearer?